jupyter-book/introduction/interoperability.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Create a randomly generated AnnData object to use as an example
</pre><p>why not use scanpy blobs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I borrowed this from another chapter. I guess blobs would also work, it doesn't really matter what the data is here. @Zethson do you have a preference?</p>

Reply:
I don't have a preference. Whatever is easiest for you

Reply:
I've kept the current data because that works but we can change it later if there is a reason to.